app/views.py

```diff
@@ -142,6 +142,34 @@ def login():
         return redirect(url_for('dashboard'))
     return render_template('login.html', title=' | Log In', form=form)
 
+@app.route('/deactivate', methods=['GET','POST'])
+def deactivate():
+    form = DeactivateForm()
+    if form.validate_on_submit():
+        user = User.query.filter_by(email=form.email.data).first()
+        if user is None:
+            flash('Invalid email. Please try again')
+            return redirect(url_for('deactivate'))
+        return redirect(url_for('deactivate'))
```

Comment:
Why is there a repeat of the same return? There is no email being sent here either.